src/core/store/modules/app/app.actions.ts

```diff
@@ -117,6 +125,47 @@ const getUserAppData = createAsyncThunk<void, { network: Network; route: Route;
   }
 );
 
+/* -------------------------------------------------------------------------- */
+/*                                  Services                                  */
+/* -------------------------------------------------------------------------- */
+
+const checkServicesStatus = createAsyncThunk<void, void, ThunkAPI>(
+  'app/checkServicesStatus',
+  async (_arg, { dispatch, extra }) => {
+    const { YEARN_ALERTS_API } = extra.config;
+    try {
+      const servicesStatusResponse = await get(`${YEARN_ALERTS_API}/health`);
+      if (servicesStatusResponse.status !== 200) throw new Error('Service status provider not currently accesible');
+
+      const errorMessageTemplate =
+        'service is currently experiencing technical issues and have been temporarily disabled. We are sorry for the inconveniences, we are working towards issues been resolved soon.';
+      const downgradedServicesMessages = [];
+      const { zapper, simulations } = servicesStatusResponse.data;
```

Comment:
Are we not getting the `notify` service here?

Reply:
Could not find any way to consume that Notify service is not working, unless the service itself fails for the user or we manually turn it off via PR unfortunately :S